src/components/definition-list/dl.component.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { StyledDl, StyledDtDiv, StyledDdDiv } from './definition-list.style';
+import Dt from './dt.component';
+import Dd from './dd.component';
+
+const Dl = ({
+  children,
+  w = 50,
+  dtTextAlign = 'right',
+  ddTextAlign = 'left'
+}) => {
+  const dtLabels = React.Children.toArray(children).filter(child => child.type === Dt);
+  const ddContent = React.Children.toArray(children).filter(child => child.type === Dd);
+  return (
+    <StyledDl data-component='dl-div'>
```

Comment:
I think data-component should just be `dl` here

Reply:
✅